src/components/Amalgamation/BusinessStatus.vue

```diff
@@ -50,6 +50,9 @@ export default class BusinessStatus extends Vue {
         return 'A BC Community Contribution Company must amalgamate to form a new BC Community ' +
           'Contribution Company.'
 
+      case AmlStatuses.ERROR_FROZEN:
+        return 'This business is frozen. Frozen businesses cannot be part of an amalgamation.'
```

Comment:
Unless this came from Yui, I recommend being consistent with the messages for Historical and PendingFiling, ie:
`This business is frozen. It cannot be part of an amalgamation.`

Reply:
As per Yui's original comment, please see second point Sev:
![second point](https://github.com/bcgov/business-create-ui/assets/122301442/7c9d7390-fc2e-4e95-a67b-7f0b0ae10474)